docs/guides/using_gpu.md

```diff
@@ -9,7 +9,7 @@ a model is compiled for CUDA, executing it on a non-CUDA-enabled machine will ra
 
 | Feature     | Built-in models | Custom Models | Deployment | DataFrame |
 | ----------- | --------------- | ------------- | ---------- | --------- |
-| GPU support | ✅               | ✅             | ✅          | ❌         |
+| GPU support | ✅              | ✅            | ✅         | ❌        |
```

Comment:
this change causes mdformat checker to fail

```
make[1]: *** [Makefile:628: check_mdformat] Error 123
```

Reply:
I reinstalled my venv, but I still get this change

Reply:
I found the problem, it was caused by a bad rebasing on main.